src/functionalTest/java/uk/gov/hmcts/reform/iacaseapi/CcdScenarioRunnerTest.java

```diff
@@ -45,16 +43,44 @@ public void setUp() {
     @Test
     public void scenarios_should_behave_as_specified() throws IOException {
 
-        for (String scenarioSource : StringResourceLoader.load("/scenarios/*.json").values()) {
+        String scenarioPattern = System.getProperty("scenario");
+        if (scenarioPattern == null) {
+            scenarioPattern = "*.json";
+        } else {
+            scenarioPattern = "*" + scenarioPattern + "*.json";
+        }
 
-            Map<String, Object> scenario = MapSerializer.deserialize(scenarioSource);
-            Map<String, String> templatesByFilename = StringResourceLoader.load("/templates/*.json");
+        Collection<String> scenarioSources =
+            StringResourceLoader
+                .load("/scenarios/" + scenarioPattern)
+                .values();
 
-            Headers authorizationHeaders = getAuthorizationHeaders(scenario);
-            String requestBody = buildCallbackRequestBody(scenario, templatesByFilename);
+        System.out.println((char) 27 + "[36m" + "-------------------------------------------------------------------");
+        System.out.println((char) 27 + "[33m" + "RUNNING " + scenarioSources.size() + " SCENARIOS");
+        System.out.println((char) 27 + "[36m" + "-------------------------------------------------------------------");
```

Comment:
I'm learning good stuff here

Reply:
I can't tell if you are being serious!
I just wanted the tests to "pop" :)